### DISCUSSION TITLE
What is the differences between nginxproxy/nginx-proxy and jwilder/nginx-proxy?

### DISCUSSION THREAD

**Discussion by akiot-b** [nginx-proxy/nginx-proxy#2280] [Q&A] [answered]
https://hub.docker.com/r/jwilder/nginx-proxy
https://hub.docker.com/r/nginxproxy/nginx-proxy
What is the difference between these?

I can't find an explanation anywhere, but it seems relevant. jwilder/nginx-proxy is out of date?

**Comment by SchoNie** [comment, marked as answer]
There is no difference, just the name. jwilder was the previous maintainer. Now buchdag is the maintainer and there was [chosen for a more common organisation he has control over. ](https://github.com/nginx-proxy/nginx-proxy/issues/1304#issuecomment-811851949)

In discussion [#2106](https://github.com/nginx-proxy/nginx-proxy/discussions/2106) you can read about the name change.

For backwards compatibility the same image is pushed to both [jwilder/nginx-proxy](https://hub.docker.com/r/jwilder/nginx-proxy) and [nginxproxy/nginx-proxy](https://hub.docker.com/r/nginxproxy/nginx-proxy) registries so people using the 'old' name did not had to change.

But the images are identical in both as you can see they have the same digest.

**Reply by akiot-b** [reply]
My questions have been answered perfectly. Thank you very much.

**Reply by buchdag** [reply]
@akiot-b in addition to @SchoNie excellent answer:

Unfortunately you can't transfer and/or rename Dockerhub registries in the same way as Github repositories, so creating new registries under an org was pretty much the only way to have something we can control (we couldn't have asked jwilder to give us control over its private Dockerhub account either, I just have a push token to jwilder/nginx-proxy and jwilder/docker-gen).

If you're curious you can take a look [at the build workflow](https://github.com/nginx-proxy/nginx-proxy/blob/main/.github/workflows/build-publish.yml#L39-L42), it's the exact same image built once and tagged with three different names (we also publish the image to Github container registry).

**Comment by akiot-b** [comment]
Close.
